styles.css

```diff
@@ -425,14 +429,14 @@ span.icone-about {
   flex-direction: column;
   align-items: center;
   gap: 20px;
-  width: 375px;
+  width: auto;
   background-color: #6070ff;
   padding: 114px 24px;
   border-top-left-radius: 75px;
 }
 
 .contact-content {
-  width: 295px;
+  width: auto;
```

Comment:
- [ ] Please kindly work on the background design to match that of [figmal](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)
**Your Design**
![yourStyle](https://user-images.githubusercontent.com/55596970/187989022-31363685-b2f7-4f57-abd4-72163db89adf.png)
**Figmal Design**
![figmal_contact](https://user-images.githubusercontent.com/55596970/187989140-752daf24-ceb5-4513-9cf7-fa65f609b85b.png)